File: src/ZKWeb.MVVMPlugins/MVVM.Common.Organization/src/Application/Dtos/UserInputDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using ZKWeb.MVVMPlugins.MVVM.Common.Base.src.Application.Dtos;

namespace ZKWeb.MVVMPlugins.MVVM.Common.Organization.src.Application.Dtos {
	[Description("用户传入信息")]
	public class UserInputDto : IInputDto {
		[Description("用户Id")]
		public Guid Id { get; set; }
		[Description("用户类型"), Required]
		public string Type { get; set; }
		[Description("用户名"), Required]
		public string Username { get; set; }
		[Description("密码")]
		public string Password { get; set; }
		[Description("租户Id")]
		public Guid OwnerTenantId { get; set; }
		[Description("备注")]
		public string Remark { get; set; }
		[Description("角色Id列表")]
		public IList<Guid> RoleIds { get; set; }

		public UserInputDto() {
			RoleIds = new List<Guid>();
		}
	}
}

File: src/ZKWeb.MVVMPlugins/MVVM.Angular.Website/static/src/modules/generated_module/dtos/user-input-dto.ts
/** 用户传入信息 */
export class UserInputDto {
	/** 用户Id */
	public Id: string;
	/** 用户类型 */
	public Type: string;
	/** 用户名 */
	public Username: string;
	/** 密码 */
	public Password: string;
	/** 租户Id */
	public OwnerTenantId: string;
	/** 备注 */
	public Remark: string;
	/** 角色Id列表 */
	public RoleIds: string[];
}
